config/initializers/location_data.rb

```diff
@@ -14,21 +14,24 @@
 
 LOCATION_POLYGON_SETTINGS = {
   regions: {
-    boundary_api: "https://ons-inspire.esriuk.com/arcgis/rest/services/Administrative_Boundaries/Regions_December_2019_Boundaries_EN_BUC/MapServer/0/query?where=1%3D1&outFields=rgn19nm,shape&outSR=4326&f=json",
-    name_key: "rgn19nm",
+    boundary_api: "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/regions/FeatureServer/0/query?where=1%3D1&outFields=GOR10NM&outSR=4326&f=json",
```

Comment:
I have reservations about how stable these endpoints are now, and what `ESMARspQHYMw9BZ9` represents/means